services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/FlowPathSwappingFsm.java

```diff
@@ -43,6 +43,8 @@
 
     protected final String flowId;
 
+    private Flow originalFlow;
```

Comment:
I've already mentioned in another PR that we didn't use Flow entity in FSMs intentionally. Multiple reasons:
- By the initial design, FSM should be serializable and FSM processor will store its state in some persistent storage. This allows to proceed with an FSM even after crash of the previous executor of the FSM. With updated solution, we should care about Flow to be serializable.  
- During FSM execution, Flow still may be amended (e.g. the path status goes DOWN). And the Flow data kept in FSM becomes obsolete. So you always need to check whether it's still up to date.
-  If we look on it, keeping YAGNI principles in mind, this brings to FSM excessive information which is not used now and requires additional efforts to handle. Why do we need these changes?